### PR TITLE
N30: country-slice

### PR DESCRIPTION
**Pull request by anopara**
Part of https://github.com/rust-gamedev/rust-gamedev.github.io/issues/912

I put it at the top of the Games section for convenience of writing. Please feel free to move it wherever editors feel it suits better 🙂

### CODE REVIEW

File: content/news/030/index.md [hunk 1]
```diff
@@ -85,6 +85,23 @@ Twitch][rust-gamedev-twitch].
...
+
+![country_slice](country_slice_7_delete_brush.gif)
+
+[Country Slice][country-slice-github] is
+[@anastasiaopara][country-slice-twitter]'s hobby project, where users can draw a
+small procedurally assembled scene.
+
+Newest addition is an erase brush and an ability to continue existing walls.
+
+Discussions: procedural in Houdini vs Rust & OpenGL 👉
```
Comment (ozkriff) [inline content/news/030/index.md:98]: The "Discussions: ..." format is usually used for public discussions so I'd rephrase that as something like "[@anastasiaopara][country-slice-twitter] also shared [a Twitter thread][country-slice-discussion] about procedural in Houdini vs Rust & OpenGL."

Comment (anopara) [reply content/news/030/index.md:98]: Sounds good 👍

**Review by 17cupsofcoffee — COMMENTED** [review]
Content is good, but could you replace the GIF with this optimized version? The current one is just over our usual file size limit, and the limited color palette means it can be compressed quite easily.

country_slice_7_delete_brush

**Review by 17cupsofcoffee — APPROVED** [review]
